### ISSUE TITLE
Plug-in

### ISSUE THREAD

**Issue by aureliencarle** [aureliencarle/cryptext#24]
I need to start right now a "plug-in" to convert browser pass word to crpytext object ... 
Maybe this plug-in will not be conserved but I need it right now for me ;)

**Comment by aureliencarle** [comment]
@GregoireUhlrich  As plug-in is left as a cryptext door for user, it can be clever to read the plug-in file from $HOME/.cryptext directory. 
HOME
   |-- pass
   |-- plug-in 
   
Let some generic plugin in plugin directory of the program an read both directory..

**Comment by GregoireUhlrich** [comment]
@aureliencarle indeed, the access using a relative path at run-time is anyway not reliable. The plugin code can be located in the cryptext directory and the input location could be provided by the user (either in the command parameters or as an `input()` command)

**Comment by aureliencarle** [comment]
.cryptext structure is now done, see #35
